e2e-tests/README.md

```diff
@@ -16,5 +16,7 @@ named `demoDelay` is provided in the `cypress.json` file. The variable can be us
 like `cy.wait(Cypress.env('demoDelay'))` to control wait times i.e. the value is the required delay in milliseconds.
 From the `e2e-tests` folder, the tests can be run locally setting the delay period with commands like:
 ```
-node_modules/cypress/bin/cypress run --headed -b chrome --env demoDelay=1000
+node_modules/cypress/bin/cypress run --headed -b chrome --env demoDelay=1000 -e TAGS="@wip"
```

Comment:
Not sure if I'm reading this wrongly, but the docs here say `-e TAGS="@wip"` but it's `-e TAGS='not @wip'` in the `package.json` - should the docs have `not` added, or the command remove `not`?

Reply:
That is correct. Here is an example of how you can select tags to run manually. The 'not @wip' in the package.json file is to avoid running wip tests in CI.